workspace_tools/project.py

```diff
@@ -72,6 +84,24 @@
         print '\n'.join(map(str, sorted(TEST_MAP.values())))
         sys.exit()
 
+    # Only prints matrix of supported IDEs
+    if options.supported_ides:
+        print mcu_ide_matrix()
+        exit(0)
+
+    # Only prints matrix of supported IDEs
+    if options.supported_ides_html:
+        html = mcu_ide_matrix(verbose_html=True)
+        f = open("./export/README.md","w")
+        try:
+            f.write("Exporter IDE/Platform Support\n")
```

Comment:
I had a look at the original README.md and this is already html code. The generated file has nearly the same format.
You can see an example in this PR, the commit 'Update README.md' is generated by this extension.